docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -944,8 +944,8 @@ The `languageSettings` block of a source set defines certain aspects of project
 kotlin {
     sourceSets.all {
         languageSettings.apply {
-            languageVersion = "1.7" // possible values: "1.4", "1.5", "1.6", "1.7"
-            apiVersion = "1.7" // possible values: "1.3", "1.4", "1.5", "1.6", "1.7"
+            languageVersion = "%kotlinVersion%" // possible values: "1.4", "1.5", "1.6", "1.7", "1.8", "1.9"
+            apiVersion = "%kotlinVersion%" // possible values: "1.3", "1.4", "1.5", "1.6", "1.7", "1.8", "1.9"
```

Comment:
I doubt you need to add `%kotlinVersion%` here. For example, `1.8.20` will be invalid value here.